config/brokerSets.json

```diff
@@ -0,0 +1,14 @@
+{
+  "brokerSets":[
```

Comment:
I think the gap is with CruiseControl not being aware of the placement logic. There could be a case where a user added brokers to the cluster and did not add them to the file. If they want to use BrokerSet isolation concept, would they want to add those brokers to the brokerSets.json and restart CC. They have an option to write their own Resolver since that is effectively an assignment policy. 

If we want to handle the "unmentioned" brokers then we can have Assignment policies like -
1) NoOpDefaultBrokerSetAssignmentPolicy.
2) ModuloDefaultBrokerSetAssignmentPolicy.

and then user can extend the interface to have their own default policy.

Reply:
Added NoOpDefaultBrokerSetAssignmentPolicy. Users can add modulo based policy and if time permits i will add it.